pandas/tseries/offsets.py

```diff
@@ -2024,7 +2024,13 @@ def __add__(self, other):
             if type(self) == type(other):
                 return type(self)(self.n + other.n)
             else:
-                return _delta_to_tick(self.delta + other.delta)
+                try:
```

Comment:
I think this might be better to change the ``_inc`` of all of the ``Tick`` and sub-classes to use ``Timedelta`` which properly handle these operations (e.g. ``Timedelta(1,'ns')`` and so on)